types/jasmine/index.d.ts

```diff
@@ -393,15 +393,15 @@ declare namespace jasmine {
 
     interface Any extends AsymmetricMatcher<any> {
         new (expectedClass: any): any;
-        jasmineToString(): string;
+        jasmineToString(pp: (value: any) => string): string;
```

Comment:
I believe we've typed pp previously in this file, so this should read as

`jasmineToString(pp: typeof pp): string;`

Reply:
Done, except that I renamed the parameter to avoid `'pp' is referenced directly or indirectly in its own type annotation`.